deployment/scaling.rst

```diff
@@ -0,0 +1,32 @@
+=========
+Scaling for Enterprise 
+=========
+
+Mattermost is designed to scale from small teams hosted on a single server to large enterprises running in cluster-based, highly available deployment configurations. 
+
+Server requirements vary based on usage and it is highly recommended that pilots are run before enterprise-wide deployments in order to estimate full scale usage based on your specific organizational needs. 
+
+Small Teams 
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Small teams can run Mattermost on a single server. 
+
+- See `install guides for step-by-step configuration instructions for single machine setup. <https://docs.mattermost.com/#install-guides>`_
+- See `hardware and software requirements for hardware sizing <https://docs.mattermost.com/install/requirements.html>`_
+
+Vertical Scaling 
```

Comment:
Calling it `Vertical Scaling` seems kind of weird.  Just usually don't see it in documentation like this since it's really "throw more hardware at it."  0/5